vespa/test_integration_docker.py

```diff
@@ -141,3 +150,125 @@ def tearDown(self) -> None:
+                type="tensor<float>(d0[32])",
+            )
+        )
+        self.app_package.schema.add_model(
+            OnnxModel(
+                model_name="bert_tiny",
+                model_file_path=os.path.join(
+                    os.getenv("RESOURCES_DIR"), "bert_tiny.onnx"
+                ),
+                inputs={
+                    "input_ids": "input_ids",
+                    "token_type_ids": "token_type_ids",
+                    "attention_mask": "attention_mask",
+                },
+                outputs={"logits": "logits"},
+            )
+        )
+        self.app_package.schema.add_rank_profile(
+            RankProfile(
+                name="bert",
+                inherits="default",
+                constants={"TOKEN_NONE": 0, "TOKEN_CLS": 101, "TOKEN_SEP": 102},
+                functions=[
+                    Function(
+                        name="question_length",
+                        expression="sum(map(query(query_token_ids), f(a)(a > 0)))",
+                    ),
+                    Function(
+                        name="doc_length",
+                        expression="sum(map(attribute(doc_token_ids), f(a)(a > 0)))",
+                    ),
+                    Function(
+                        name="input_ids",
+                        expression="tensor<float>(d0[1],d1[128])(\n"
+                        "    if (d1 == 0,\n"
+                        "        TOKEN_CLS,\n"
+                        "    if (d1 < question_length + 1,\n"
+                        "        query(query_token_ids){d0:(d1-1)},\n"
+                        "    if (d1 == question_length + 1,\n"
+                        "        TOKEN_SEP,\n"
+                        "    if (d1 < question_length + doc_length + 2,\n"
+                        "        attribute(doc_token_ids){d0:(d1-question_length-2)},\n"
+                        "    if (d1 == question_length + doc_length + 2,\n"
+                        "        TOKEN_SEP,\n"
+                        "        TOKEN_NONE\n"
+                        "    ))))))",
+                    ),
+                    Function(
+                        name="attention_mask",
+                        expression="map(input_ids, f(a)(a > 0)) ",
+                    ),
+                    Function(
+                        name="token_type_ids",
+                        expression="tensor<float>(d0[1],d1[128])(\n"
+                        "    if (d1 < question_length,\n"
+                        "        0,\n"
+                        "    if (d1 < question_length + doc_length,\n"
+                        "        1,\n"
+                        "        TOKEN_NONE\n"
+                        "    )))",
+                    ),
+                    Function(
+                        name="eval",
+                        expression="tensor(x{}):{x1:onnxModel(bert_tiny).logits{d0:0,d1:0}}",
+                    ),
```

Comment:
Can be simplified now to `onnx(bert_tiny).logits{d0:0,d1:0}`. `onnx` and `onnxModel` are now synonymous. Actually, you don't need this `eval` funciton at all, can just put it directly in second phase.

Reply:
I tried to change, and I get the following message: 
```
WARNING: invalid rank feature 'onnx(bert_tiny).logits': unknown basename: 'onnx'\n
```
I thought this might be because I had an old Vespa image, but the same error is displayed when I deploy to hosted vespa.